app/services/users/profile.rb

```diff
@@ -96,11 +96,20 @@ def claims
           military_history: Vet360Policy.new(user).military_access?,
           payment_history: BGSPolicy.new(user).access?(log_stats: false),
           personal_information: MPIPolicy.new(user).queryable?,
-          rating_info: LighthousePolicy.new(user).rating_info_access?
+          rating_info: LighthousePolicy.new(user).rating_info_access?,
+          **form_526_required_identifiers
         }
       end
     end
 
+    def form_526_required_identifiers
+      if Flipper.enabled?(:form_526_required_identifiers_in_user_object, user)
+        return { form526_required_identifier_presence: Users::Form526UserIdentifiersStatusService.call(user) }
+      end
+
+      {}
+    end
```

Comment:
The following more-idiomatic alternative should help simplify future code deletion.
```
return {} unless Flipper.enabled?(:form_526_required_identifiers_in_user_object, user)
{ form526_required_identifier_presence: Users::Form526UserIdentifiersStatusService.call(user) }
```